src/models/comments_model.ts

```diff
@@ -0,0 +1,28 @@
+import mongoose, { Document, Schema, Types, ObjectId } from "mongoose";
+
+export interface CommentDocument extends Document {
```

Comment:
```suggestion
export interface ICommentDocument extends Document {
```

As far as I know, the naming conventions for interfaces is starting with I